tests/util/test_yaml.py

```diff
@@ -90,13 +93,18 @@ def test_include_dir_list(self, mock_walk):
     def test_include_dir_list_recursive(self, mock_walk):
         """Test include dir recursive list yaml."""
         mock_walk.return_value = [
-            ['/tmp', ['tmp2'], ['zero.yaml']],
+            ['/tmp', ['tmp2', '.ignore', 'ignore'], ['zero.yaml']],
             ['/tmp/tmp2', [], ['one.yaml', 'two.yaml']],
+            ['/tmp/.ignore', [], ['ignore.yaml']],
```

Comment:
This mock is bad. Simce you modify dirs in place the real os.walk will skip this folder. This mock blindly returns an invalid directory